File: src/main/java/santa/seedcopy/SeedCopyLogger.java
package santa.seedcopy;

public class SeedCopyLogger {

    /**
     * Prints a log message if the enableLogging config option is true.
     * @param message The message to print.
     */
    public static void printLogMessage(String message) {
        if (Config.enableLogging) {
            System.out.println(message);
        }
    }
}

File: src/main/kotlin/santa/seedcopy/SeedCopyLogger.kt
package santa.seedcopy

object SeedCopyLogger {
    /**
     * Prints a log message if the enableLogging config option is true.
     * @param message The message to print.
     */
    @JvmStatic
    fun printLogMessage(message: String) {
        if (Config.ENABLE_LOGGING) {
            println(message)
        }
    }
}